### ISSUE TITLE
Gallery visuals

### ISSUE THREAD

**Issue by martinamorris** [EpiModel/EpiModel-Gallery#28]
It would be great if the gallery were really a gallery -- with visuals and brief descriptions of each example.  Maybe a website landing page?  And a wiki for FAQ and other info?  Could be built up over time...

**Comment by smjenness** [comment]
Agreed! Each example has a brief description in the READMEs but these could be expanded. It would be great to have help with this, from your UW group.

**Comment by martinamorris** [comment]
I'm not sure we have the capacity for this.  Jeanette has the right expertise, but she's out of service right now given family responsibilities during Covid.
